akd/src/storage/mod.rs

```diff
@@ -151,7 +151,7 @@ pub trait Storage: Clone {
         &self,
         keys: &[types::AkdLabel],
```

Comment:
Not related to your change, but could we rename `keys` to `usernames` to be consistent with `get_user_data` and `get_user_state`?